bundles/org.openhab.binding.froniuswattpilot/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,210 @@
+	</channel-group-type>
+
+	<channel-group-type id="status">
+		<label>Charging Status</label>
+		<channels>
+			<channel id="charging-state" typeId="charging-state"/>
+			<channel id="charging-possible" typeId="charging-possible"/>
+			<channel id="single-phase" typeId="charging-single-phase"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="metrics">
+		<label>Charging Metrics</label>
+		<channels>
+			<channel id="power" typeId="system.electric-power">
+				<label>Total Power</label>
+			</channel>
+			<channel id="energy-session" typeId="system.electric-energy">
+				<label>Charged Energy</label>
+				<description>Energy charged in the current/last session</description>
+			</channel>
+			<channel id="energy-total" typeId="system.electric-energy">
+				<label>Total Charged Energy</label>
+				<description>Energy charged in total</description>
+			</channel>
+			<channel id="l1-power" typeId="system.electric-power">
+				<label>Phase 1 Power</label>
+			</channel>
+			<channel id="l2-power" typeId="system.electric-power">
+				<label>Phase 2 Power</label>
+			</channel>
+			<channel id="l3-power" typeId="system.electric-power">
+				<label>Phase 3 Power</label>
+			</channel>
+			<channel id="l1-voltage" typeId="system.electric-voltage">
+				<label>Phase 1 Voltage</label>
+			</channel>
+			<channel id="l2-voltage" typeId="system.electric-voltage">
+				<label>Phase 2 Voltage</label>
+			</channel>
+			<channel id="l3-voltage" typeId="system.electric-voltage">
+				<label>Phase 3 Voltage</label>
+			</channel>
+			<channel id="l1-current" typeId="system.electric-current">
+				<label>Phase 1 Current</label>
+			</channel>
+			<channel id="l2-current" typeId="system.electric-current">
+				<label>Phase 2 Current</label>
+			</channel>
+			<channel id="l3-current" typeId="system.electric-current">
+				<label>Phase 3 Current</label>
+			</channel>
+		</channels>
+	</channel-group-type>
+
+	<!-- Control Channel Types -->
+	<channel-type id="charging-allowed">
+		<item-type>Switch</item-type>
+		<label>Charging Allowed</label>
+		<description>Allow or forbid charging</description>
+		<category>BatteryLevel</category>
+		<tags>
+			<tag>Control</tag>
+		</tags>
+		<state>
+			<options>
+				<option value="ON">Allow charging</option>
+				<option value="OFF">Forbid charging</option>
+			</options>
+		</state>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+	<channel-type id="charging-mode">
+		<item-type>String</item-type>
+		<label>Charging Mode</label>
+		<description>The mode of charging</description>
+		<category>BatteryLevel</category>
+		<tags>
+			<tag>Control</tag>
+		</tags>
+		<state>
+			<options>
+				<option value="DEFAULT">Default</option>
+				<option value="ECO">Eco</option>
+				<option value="NEXT_TRIP">Next Trip</option>
+			</options>
+		</state>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+	<channel-type id="charging-current">
+		<item-type>Number:ElectricCurrent</item-type>
```

Comment:
I don't remember system default unit for current, but in case it's not A:
```suggestion
		<item-type unitHint="A">Number:ElectricCurrent</item-type>
```

Reply:
Ampere (A) is the default unit, just checked the docs :+1: